test/one_dragon_qt/logic/test_image_analysis_logic.py

```
@@ -0,0 +1,49 @@
+    logic = _create_logic()
+
+    name = logic.auto_save_pipeline()
+
+    assert name == '流水线1'
+    assert logic.active_pipeline_name == '流水线1'
+    logic.cv_service.get_pipeline_names.assert_called_once_with(source='')
+    logic.cv_service.save_pipeline.assert_called_once_with('流水线1', logic.pipeline, source='')
+
+
+def test_auto_save_pipeline_skips_existing_names():
+    """已存在「流水线1」「流水线2」时，自动保存为「流水线3」，不覆盖已有文件"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)